packages/core/src/components/way-select/way-select.tsx

```diff
@@ -0,0 +1,245 @@
+import { Component, Method, Element, Event, EventEmitter, Listen, State, Prop, Watch, h } from '@stencil/core';
+import { IFormElementData } from '../../types/form';
+import Fragment from '../way-fragment/way-fragment';
+import { ISelectOption, ISelectConfig } from '../../types/select';
+
+@Component({
+  tag: 'way-select',
+  styleUrl: 'way-select.scss',
+  shadow: true,
+})
+export class W2wSelect {
```

Comment:
done

Reply:
Seems you didn't read me whole comment? We decided to make a more regular way-select component, which is not this component. And that this one would be called WayAutosuggest instead of WaySelect.

Reply:
Yeah, I don't know what is happening but this (and a lot of other stuff) is outdated. If you look at the latest code. It should be renamed to way-autosuggest.

Reply:
Ah sorry, I thought I had the latest changes, but didn't apparently.